app/models/post_action_type.rb

```diff
@@ -67,7 +68,16 @@ def flag_types_without_custom
 
     def flag_types
       return flag_settings.flag_types if overridden_by_plugin_or_skipped_db?
-      flag_enum(all_flags)
+
+      # Once replace_flag API is fully deprecated, then we can drop respond_to. It is needed right now for migration to be evaluated.
```

Comment:
Please put this + the other comment as a `TODO (lis2/krisk)` so we know who is responsible :)